coretex/entities/dataset/network_dataset.py

```diff
@@ -39,6 +40,7 @@
 
 
 SampleType = TypeVar("SampleType", bound = "NetworkSample")
+NAME_VALIDATION_MESSAGE = ">> [Coretex] Sample name is invalid. Requirements: alphanumeric characters (\"a-z\", and \"0-9\") and dash (\"-\") with length between 3 to 50"
```

Comment:
This message addresses Sample's only, but is being raised for Dataset as well. Can we update it to say "Entity name is invalid......"